### PR TITLE
Updated Switch guidelines

### LINKED ISSUE #2891: Switcher: Update guidelines to show ability to have a check mark in any switcher variation
Update images with new disabled styling + add ability to add checkmark to any switch variation

### PR DESCRIPTION
**Pull request by doruskova**
I've updated the images and described the switch with a checkmark and label. 
Images in Sketch Docs assets have been updated. 

Closes #2891

### CODE REVIEW

File: packages/v4/src/content/design-guidelines/components/switch/switch.md [hunk 1]
```diff
@@ -8,11 +8,12 @@ A **switch** toggles the state of a setting (between on and off). Switches and [
...
+<img src="./img/switch.png" alt="A side-by-side comparison of a basic switch and a switch without a label. The basic switch uses both a form label and a switch label, while the switch without a label uses a form label and replaces the switch label with a checkmark inside the switch itself" width="792"/>
 
 1. **Form label:** Indicates what the switch turns on or off
 2. **Switch label:** Indicates whether the switch itself is turned on or off
 3. **Switch without label:** Indicates the state of a switch with a checkmark inside the switch itself, used in place of a switch label
+4. **Switch with label and checkmark:** Indicates the state of a switch with a checkmark inside the switch itself and label with additional message 
 
 Both switch types display their state (on or off) through different methods and locations. A basic switch communicates its state with an exterior switch label added to its right, while a switch without a label uses a checkmark to communicate its state from inside the switch itself.
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/switch/switch.md:18]: Change this to say "All switch types" instead of "Both"

Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/switch/switch.md:18]: Change this to say "All switch types" instead of "Both"

**Review by mmenestr — COMMENTED** [review]
Just have one small comment, otherwise looks good!

**Comment by doruskova** [comment]
Updated :)

**Review by mmenestr — APPROVED** [review]

**Pull request merged**